tests/lib/pkgdb.sh

```diff
@@ -441,6 +441,12 @@ distro_install_build_snapd(){
         apt install -y --only-upgrade snapd
         mv sources.list.back /etc/apt/sources.list
         apt update
+
+        # In case the PPA does not have an updated snapd the apt install snapd call will not
+        # error and we only test the distro snapd. So we make apt show to double check the
+        # snapd package comes from a ppa
+        apt show snapd | grep "APT-Sources: http.*ppa.launchpad.net"
```

Comment:
This looks good - but also unrelated to this PR? Did this accidentally slip into this PR?

Reply:
yes it is accidental, It is already on master, perhaps it was introduced on the initial marge.
But I think is shouldn't affect due to this is already merged on master https://github.com/snapcore/snapd/blob/master/tests/lib/pkgdb.sh#L448

Reply:
When I expand the diff to see more context I see that this ppa is added under ` if [ "$SRU_VALIDATION" = "1" ]; then`. When expanding below I see ` elif [ -n "$PPA_VALIDATION_NAME" ]; then` and the PPA check is there too. So it seems like this will add it again in the wrong place (in the SRU_VALIDATION path)? Or am I misreading this?

Reply:
Well what you found is that the check was duplicated and in the incorrect place. I just deleted that and if you look few lines below you will see the same check but in the ppa section (which is correct).
Thanks for finding this.